types/istanbul-reports/istanbul-reports-tests.ts

```diff
@@ -57,3 +58,6 @@ create('text-lcov', { projectRoot: 'foo' });
 
 create('text-summary');
 create('text-summary', { file: 'foo' });
+
```

Comment:
why do you want to test this? It's external to this module (and non essential IMO)
I"ve rather test if returned instance matches base contract

Reply:
Because the `ReportBase` return type does not come from this package, but from `istanbul-lib-report`. Therefore, I think it makes sense to assert that the type returned behaves as it would be used with the `report` package. This is the real world code snippet from the package: https://github.com/istanbuljs/istanbuljs/tree/master/packages/istanbul-lib-report